packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -252,7 +257,22 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 		<Stack>
 			<StackItem>
 				<Table size="extra-small" ref={tableRef}>
-					{DataRow("Container", containerState.id, columnSizing_unstable)}
+					{DataRow(
+						<>
+							{"Container ID "}
+							<TooltipHost
+								content="Note that in Telemetry this value will show up as 'docId', and 'containerId' is something different."
```

Comment:
We should also probably define what the `containerId` is, in addition to noting the terminology mismatch.